CHANGELOG.rst

```diff
@@ -6,6 +6,9 @@ Changelog
 
 .. note:: This version is not yet released and is under active development.
 
+* Switched to the new `cffi`_ ``set_source`` out-of-line API mode for
+  compilation. This results in significantly faster imports and lowered
+  memory consumption.
```

Comment:
I hate to trigger another test run... but I feel like this should mention that this means we no longer support PyPy releases older than 2.6 nor do we support any released version of PyPy3 and won't until one is released that supports CFFI 1.0+.